cmd/cli/kubectl-kyverno/utils/common/common.go

```diff
@@ -991,7 +989,7 @@ func GetPatchedResourceFromPath(fs billy.Filesystem, path string, isGit bool, po
 			if err != nil {
 				fmt.Printf("Unable to open patchedResource file: %s. \nerror: %s", path, err)
 			}
-			patchedResourceBytes, err = ioutil.ReadAll(filep)
+			patchedResourceBytes, _ = ioutil.ReadAll(filep)
```

Comment:
What could happen if we don't check the error?

Reply:
No idea TBH

Reply:
Will do asap

Reply:
Done